### ISSUE TITLE
Retry download and unzip of KoreBuild

### ISSUE THREAD

**Issue by mikeharder** [aspnet/KoreBuild#223]
CI build recently failed with error downloading and unzipping KoreBuild:

```
  ============ Building Microsoft.Data.Sqlite ============
  Downloading KoreBuild from https://github.com/aspnet/KoreBuild/archive/dev.zip
  [/tmp/KoreBuild-D83ECDEA-D727-4C70-9357-1468C00855DF/korebuild.zip]
    End-of-central-directory signature not found.  Either this file is not
    a zipfile, or it constitutes one disk of a multi-part archive.  In the
    latter case the central directory and zipfile comment will be found on
    the last disk(s) of this archive.
  unzip:  cannot find zipfile directory in one of /tmp/KoreBuild-D83ECDEA-D727-4C70-9357-1468C00855DF/korebuild.zip or
          /tmp/KoreBuild-D83ECDEA-D727-4C70-9357-1468C00855DF/korebuild.zip.zip, and cannot find /tmp/KoreBuild-D83ECDEA-D727-4C70-9357-1468C00855DF/korebuild.zip.ZIP, period.
  cp: /tmp/KoreBuild-D83ECDEA-D727-4C70-9357-1468C00855DF/**/build/**: No such file or directory
  chmod: .build/KoreBuild.sh: No such file or directory
  ./build.sh: line 46: .build/KoreBuild.sh: No such file or directory
/Users/asplab/Documents/buildAgent/work/2b0c8ce712895332/build/RepositoryBuild.targets(60,5): error MSB3073: The command "./build.sh /t:Verify /p:BuildNumber=19317 /p:Configuration=Release" exited with code 127.
```

http://aspnetci/viewLog.html?buildId=200007&buildTypeId=XPlat_Mac_MacUniverseCoreCLR&tab=buildLog

If unzip fails, `build.sh` should retry both the download and unzip commands.

**Comment by Eilon** [comment]
Moving to 2.1.0.

**Comment by natemcmaster** [comment]
Closing this now that we no longer download KoreBuild from github.com as a zip.

Resolved along with the work in aspnet/KoreBuild#258.
